### PR TITLE
[camera] change property of `FlutterTextureRegistry` from strong to weak

### PR DESCRIPTION
**Pull request by werainkhatri**
Fixes: https://github.com/flutter/flutter/issues/102966

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

File: packages/camera/camera/ios/Classes/FLTThreadSafeTextureRegistry.m [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #import "QueueUtils.h"
 
 @interface FLTThreadSafeTextureRegistry ()
-@property(nonatomic, strong) NSObject<FlutterTextureRegistry> *registry;
+@property(nonatomic, weak) NSObject<FlutterTextureRegistry> *registry;
```
Comment (hellohuanlin) [inline packages/camera/camera/ios/Classes/FLTThreadSafeTextureRegistry.m:9]: If we want to release the texture registry we should release the whole wrapper object.

**Review by hellohuanlin — CHANGES_REQUESTED** [review]

File: packages/camera/camera/ios/Classes/FLTThreadSafeTextureRegistry.m (hunk 1 above)
Comment (werainkhatri) [reply packages/camera/camera/ios/Classes/FLTThreadSafeTextureRegistry.m:9]: sorry for the delay.
didn't get you here (fairly a beginner at ObjC)

**Comment by Hixie** [comment]
@werainkhatri  Will you be able to continue working on this? It's totally fine if not; we can take it over (though it might take longer). I'm just asking because I'm trying to clean out our review queue.

**Comment by werainkhatri** [comment]
i don't think I'll be able to. i don't seem to be qualified enough 😅. closing...

**Pull request closed without merging**